drivers/syslog/syslog_intbuffer.c

```diff
@@ -53,79 +54,71 @@
 
 /* This structure encapsulates the interrupt buffer state */
 
-struct g_syslog_intbuffer_s
+struct syslog_intbuffer_s
 {
-  volatile uint16_t si_inndx;
-  volatile uint16_t si_outndx;
-  uint8_t si_buffer[CONFIG_SYSLOG_INTBUFSIZE];
+  struct circbuf_s circ;
+  spinlock_t       splock;
+  uint8_t          buffer[CONFIG_SYSLOG_INTBUFSIZE];
 };
 
 /****************************************************************************
  * Private Data
  ****************************************************************************/
 
-static struct g_syslog_intbuffer_s g_syslog_intbuffer;
+static struct syslog_intbuffer_s g_si_buffer =
```

Comment:
why not keep g_syslog_intbuffer

Reply:
Too long too complex

Reply:
so do you plan to change all syslog_intbuffer to syslog_si?

Reply:
But this is just an internal variable, and the concise abbreviation will avoid coding style issues, especially without adding too many line breaks.

Reply:
Even internal variable should keep the name consistency too.

Reply:
Let me guess, do you have an internal script that hard codes this variable, so you don't want to make similar changes?

Reply:
You could approve such a naming method. Why do you order others that they must follow your style?
https://github.com/apache/nuttx/pull/15092/files#diff-685cad5d9c2ca106337a0796fa6df48505ccf6424220f42afc4703a485b88cb0R116

![image](https://github.com/user-attachments/assets/bad2ab74-f5a0-47d0-8884-2b47f372ebc7)

could I use `buffer` directly instead of `syslog_intbuffer`?

Reply:
I think this problem is fixed with the new patch.

Reply:
already fixed !

Reply:
ditto https://github.com/apache/nuttx/pull/15094 :
https://github.com/apache/nuttx/commit/1e47441775bebd05e872d3ba8ad6d8a76169ed96
https://github.com/apache/nuttx/blob/master/arch/arm/src/max326xx/max32660/max32660_rtc.c#L83

Reply:
@hujun260 please fix it.

Reply:
https://github.com/apache/nuttx/pull/15229